### PR TITLE
Add papercolor dark variant

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**
Currently looks like this :smile: 

image

### CODE REVIEW

**Comment by uncomfyhalomacro** [comment]
Ready for review. Thank you! :smile:

File: runtime/themes/papercolor-dark.toml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+"ui.background" = {bg="background"}
+"ui.text" = "foreground"
+"ui.text.focus" = { fg = "selection_background", modifiers = ["bold"]}
+"ui.selection" = {bg="selection_background", fg="selection_foreground"}
+"ui.cursorline" = {bg="cursorline_background"}
+"ui.statusline" = {bg="paper_bar_bg", fg="regular0"}
+"ui.statusline.select" = {bg="background", fg="bright7"}
+"ui.statusline.normal" = {bg="background", fg="bright3"}
+"ui.statusline.inactive" = {bg="background", fg="bright0"}
+"ui.virtual" = "indent"
```
Comment (the-mikedavis) [inline runtime/themes/papercolor-dark.toml:14]: It looks like `indent` isn't defined in the palette

**Review by the-mikedavis — APPROVED** [review]
Looks good, thanks! 🎨

**Comment by the-mikedavis** [comment]
Could you add a screenshot of this to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Pull request merged**